src/content/app/tools/blast/state/blast-form/blastFormSlice.ts

```diff
@@ -23,12 +23,14 @@ export type BlastFormState = {
   step: 'sequences' | 'species'; // will only be relevant on smaller screens
   sequences: ParsedInputSequence[];
   shouldAppendEmptyInput: boolean;
+  selectedSpecies: { [genomeId: string]: boolean };
```

Comment:
It should probably be an array of strings (selected genome ids) rather than an object keyed by genome ids.

Reply:
I made it an object because we might need other species attributes to pass to the tools API.

Reply:
Converted into an array. Done